File: programacao_orientada/cadastro_funcionario_java/src/funcionario.java
import java.util.Scanner;

public class funcionario extends pessoa{
    private double salario;

    public funcionario(String nome, int idade, double salario) {
        super(nome, idade);
        this.salario = salario;
    }

    public funcionario() {
        Scanner scan = new Scanner(System.in);
        System.out.println("\nDigite o nome do funcionario: ");
        setNome(scan.nextLine());
        System.out.println("\nDigite a idade: ");
        setIdade(scan.nextInt());
        System.out.println("\nDigite o salario: ");
        setSalario(scan.nextInt());
        scan.nextLine();
    }

    public double getSalario() {
        return salario;
    }


    public void setSalario(double salario) {
        this.salario = salario;
    }

    public void executarTarefa(){
        System.out.println(getNome() + " está executando uma tarefa genérica\n");
    }
}

File: programacao_orientada/cadastro_funcionario_python/classes/Funcionario.py
from classes.Pessoa import Pessoa

class Funcionario(Pessoa):
    def __init__(self):
        super().__init__()
        
        print("\nDigite o nome do funcionario: ")
        self.nome = input()
        
        print("\nDigite a idade: ")
        self.idade = int(input())
        
        print("\nDigite o salario: ")
        self.salario = float(input()) 
    
    def get_salario(self):
        return self.salario

    def executar_tarefa(self):
        print(f"{self.nome} está executando uma tarefa genérica\n")

        